res/values/strings.xml

```diff
@@ -307,24 +307,24 @@
     <string name="ConversationTitleView_verified">Verified</string>
 
     <!-- CreateProfileActivity -->
-    <string name="CreateProfileActivity_your_profile_info">Your profile info</string>
+    <string name="CreateProfileActivity_your_profile_info">Your profile information</string>
     <string name="CreateProfileActivity_error_setting_profile_photo">Error setting profile photo</string>
     <string name="CreateProfileActivity_problem_setting_profile">Problem setting profile</string>
     <string name="CreateProfileActivity_profile_photo">Profile photo</string>
     <string name="CreateProfileActivity_too_long">Too long</string>
     <string name="CreateProfileActivity_profile_name">Profile Name</string>
     <string name="CreateProfileActivity_set_up_your_profile">Set up your profile</string>
-    <string name="CreateProfileActivity_signal_profiles_are_end_to_end_encrypted">Signal profiles are end-to-end encrypted, and the Signal service never has access to this information.</string>
+    <string name="CreateProfileActivity_signal_profiles_are_end_to_end_encrypted">Your Signal Profile will be visible only to people who are in your contact list, to people who you intitiated a conversation with, and to people for who you explicitly gave permission by tapping on the banner \'make my profile visible\'. Signal profiles are end-to-end encrypted, so even the Signal service never has access to your profile information.</string>
```

Comment:
I added crucial information here about who can and who can not see your profile. I assume many users want to know this but can't find that information anywhere else in the app. It becomes long, but it fits with ease on the screen. I removed the word information because you requested so Valldrac, but in my opinion the word information does fit well.